controllers/progressivesync_controller_test.go

```diff
@@ -81,13 +81,13 @@ var _ = Describe("ProgressiveRollout Controller", func() {
 
 		It("should forward events for owned applications", func() {
 			By("creating an owned application")
-
-			ownerPR := createOwnerPR(namespace, "owner-pr")
+			localNS := namespace
```

Comment:
the alternative to the local copy is to create the `types.NamespacedName` comparison variable _before_ the `Eventually`